components/calculation/PrettyOutput.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Collapse, Paper } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import { StatementResult } from 'fqm-execution';
+
+export interface PrettyOutputProps {
+  statement: StatementResult | undefined;
+}
+
+export default function PrettyOutput({ statement }: PrettyOutputProps) {
+  const [opened, { toggle }] = useDisclosure(false);
+  const buttonText = opened ? 'Hide Results' : 'Show Results';
```

Comment:
Just a question. Why put the button text here instead of in the TSX stuff below?

Reply:
Honestly, I am not sure 😆 But I am going to move it in the TSX stuff below because I agree that makes more sense.